README.md

```diff
@@ -300,9 +300,9 @@ Spokestack.onSuccess = e => {
 | Spokestack.onTimeout(event)          | `null`                | Invoked when no speech has been detected for `wake-active-max` after activation                                      | Android, iOS |
 | Spokestack.onTrace(event)            | `message`:`string`    | Invoked when a trace message become available                                                                        | Android      |
 | Spokestack.onError(event)            | `error`:`string`      | Invoked upon an error in the speech pipeline execution                                                               | Android, iOS |
-| Spokestack.onSuccess(event)          | `url`:`string`   | Invoked upon a successful TTS synthesis request                | iOS          |
-| Spokestack.onFailure(event)          | `error`:`string` | Invoked upon a failed TTS synthesis request                    | iOS          |
-| Spokestack.onClassification(event) | `result`:`dictionary` | Invoked upon a successful NLU utterance classification | iOS          |
+| Spokestack.onSuccess(ttsEvent)          | `url`:`string`   | Invoked upon a successful TTS synthesis request                | iOS          |
+| Spokestack.onFailure(ttsEvent)          | `error`:`string` | Invoked upon a failed TTS synthesis request                    | iOS          |
+| Spokestack.onClassification(nluEvent) | `result`:`dictionary` | Invoked upon a successful NLU utterance classification | iOS          |
```

Comment:
The way these events are documented is strange to me. These all look like function calls where the user is responsible for passing an event. Wouldn't it be the Android equivalent of `Spokestack.onSuccess((event) => {})` instead?

Reply:
not sure what you mean here. are you complaining that the `nluEvent`/`ttsEvent` nomenclature leads you to think of them as functions, whereas `event` lets you think of them as events? This part of the doc doesn't have anything to do with Android, it was just an update to how the event args were named to provide additional clarity.

Reply:
This can be changed in a subsequent PR. I'm saying that the way it reads does not currently make sense to me.

Reply:
ok. feel free to open an issue that explains a little bit and I'll address in a subsequent PR.